src/main/java/it/gov/pagopa/cgn/portal/repository/EycaDataExportRepository.java

```diff
@@ -5,6 +5,6 @@
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 
 public interface EycaDataExportRepository
-        extends JpaRepository<EycaDataExportViewEntity, String>, JpaSpecificationExecutor<EycaDataExportViewEntity> {
+        extends JpaRepository<EycaDataExportViewEntity, Long>, JpaSpecificationExecutor<EycaDataExportViewEntity> {
```

Comment:
why this change in the generic?

Reply:
Same fix as the other but this time EycaDataExportViewEntity.id is Long not String thus the type of ID in generic repository must be the same.